pkg/shared/kafkautil/dedicated_utils.go

```diff
@@ -0,0 +1,26 @@
+package kafkautil
+
+import (
+	"context"
+	"github.com/redhat-developer/app-services-cli/pkg/shared/factory"
+	kafkamgmtclient "github.com/redhat-developer/app-services-sdk-core/app-services-sdk-go/kafkamgmt/apiv1/client"
+)
+
+func ListEnterpriseClusters(f *factory.Factory) (*kafkamgmtclient.EnterpriseClusterList, error) {
```

Comment:
`shared utils` is a place where methods defined are used in different subcommands. For example - `kafkautil.InteractiveSelect` is used in `kafka`, `connectors` and `cluster` subcommands.
`ListEnterpriseClusters` is used just in dedicated list, will it make sense to move it to `pkg/cmd/dedicated/dedicatedcmdutil/dedicated_util.go`?

Reply:
it will also be used in the `create kafka` cmd too, so that's the reason I moved it to shared